### ISSUE TITLE
Not ready to Production

### ISSUE THREAD

**Issue by Ben621** [shakedzy/dython#66]
HI,
First, I would like to say, thank you that you have implemented this library and also the blog you have written [The Search for Categorical Correlation](https://towardsdatascience.com/the-search-for-categorical-correlation-a1cf7f1888c9)
Second:
- the standard of python package is coverage of tests.
you can read [python.doc](https://docs.python-guide.org/writing/tests/)
- that means that current code I not production-ready.
- you should use tox, pytest, mypy
Therefore I will show you why it's important:

### Version check:

Run and copy the output:
```python
import sys

print(sys.version_info)

sys.version_info(major=3, minor=8, micro=1, releaselevel='final', serial=0)

print(dython.__version__)

0.6.1
```

### Describe the bug:
pandas.DataFrame.as_matrix has been deprecades see [pandas doc](https://pandas.pydata.org/pandas-docs/version/0.23.4/generated/pandas.DataFrame.as_matrix.html#pandas-dataframe-as-matrix)
Deprecated since version 0.23.0: `Use DataFrame.values() instead`.

Code to reproduce:
```python
import dython

import pandas as pd

pd.DataFrame(data={'col1': [1, 2]})

   col1

0     1

1     2

df = pd.DataFrame(data={'col1': [1, 2]})

dython._private.convert(df, 'array')

Traceback (most recent call last):

  File "<input>", line 1, in <module>

  File "C:\Users\AS6SE\PycharmProjects\omri_code\venv\lib\site-packages\dython\_private.py", line 15, in convert

    converted = data.as_matrix()

  File "C:\Users\AS6SE\PycharmProjects\omri_code\venv\lib\site-packages\pandas\core\generic.py", line 5139, in __getattr__

    return object.__getattribute__(self, name)

AttributeError: 'DataFrame' object has no attribute 'as_matrix'
```

## Error message:

Error message:
```
Traceback (most recent call last):

  File "<input>", line 1, in <module>

  File "C:\Users\AS6SE\PycharmProjects\omri_code\venv\lib\site-packages\dython\_private.py", line 15, in convert

    converted = data.as_matrix()

  File "C:\Users\AS6SE\PycharmProjects\omri_code\venv\lib\site-packages\pandas\core\generic.py", line 5139, in __getattr__

    return object.__getattribute__(self, name)

AttributeError: 'DataFrame' object has no attribute 'as_matrix'
```

**Comment by shakedzy** [comment]
@Ben621 so I understand you'll be writing these tests? Great. Tag me on your PR.

**Comment by Ben621** [comment]
you didn't solve the issue.
you still use `data.as_matrix()` is depricated.
in line https://github.com/shakedzy/dython/blob/eed757172fd58209bec353911b7301b9def7db32/dython/_private.py#L15

**Comment by shakedzy** [comment]
Hi @Ben621 , let's talk a little about what you wrote here:

- **The package is note ready for production**: it's not meant for production, it actually says so on the documentation homepage.
- **you should use tox, pytest, mypy**: I asked you to write these tests, as I have no intention to do so at the moment. You didn't reply, so I'll take it as a no

The like of code you point is never used, which is why this error never occurred. It remains there till there will be tests, to verify they work. If you feel like participating, please contribute to issue #69. 

Bottom line, you said this project is not production ready, and you're right. It wasn't meant to. Therefore, issue is closed.
